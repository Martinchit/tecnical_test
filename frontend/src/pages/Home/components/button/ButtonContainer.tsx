import styled from 'styled-components';
import { Button } from 'react-bootstrap';

export const ButtonContainer = styled(Button)`
  background-color: #fff;
  border: none;
  &:hover {
    outline: none;
    box-shadow: none;
    background-color: #fff;
  }
  &:focus {
    outline: none;
    box-shadow: none;
    background-color: #fff;
  }
`;
