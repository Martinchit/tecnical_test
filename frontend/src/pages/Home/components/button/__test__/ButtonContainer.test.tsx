import React from 'react';
import { shallow } from 'enzyme';

import { ButtonContainer } from '../ButtonContainer';

describe('ButtonContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ButtonContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
