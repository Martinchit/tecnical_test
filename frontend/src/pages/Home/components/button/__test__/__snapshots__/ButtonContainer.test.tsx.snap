// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonContainer renders correctly 1`] = `
<Button
  active={false}
  className="sc-AxjAm jRkGQ"
  disabled={false}
  type="button"
  variant="primary"
/>
`;
